File: 4st/Hani/programmer/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        int max = 0;
        int min = 0;
        if (a > b) {
            max = a;
            min = b;
        }
        else if (a < b){
            max = b;
            min = a;
        }
        else if (a == b) {
            answer = a;
        }
        for (int i = min; i <= max; i++) {
                answer += i;
        }
        return answer;
    }
}

File: 4st/Hani/programmer/두 정수 사이의 합.py
def solution(a, b):
    answer = 0
     
    if a < b :
        for i in range(a, b+1) :
            answer += i
    elif a > b :
        for i in range(b, a+1) :
            answer += i
    else :
        answer = a
        
    return answer